ladybug_comfort/windchill.py

```diff
@@ -0,0 +1,164 @@
+# coding=utf-8
+"""Utility functions for calculating Wind Chill Index (WCI) and Wind Chill
+Temperature (WCT)"""
+
+import math
+
+
+def windchill_index(ta, ws):
+    """Calculate the Wind Chill Index (WCI) from air temperature  and wind
+    speed.
+
+    Wind Chill Index is derived from original work carried out by Gregorczuk[1].
+    It qualifies thermal sensations of man in wintertime. It is especially
```

Comment:
I think we can remove `of man` here in the docstring (it incorrectly implies that the index is not applicable to women). I think this phrase was a bit lost in translation from the original work Djordje was pulling from.